### PR TITLE
MOSIP-37256 : Updated readme for developer guide java21

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,14 @@
...
 ## Overview
-This repository contains modules used for audit related funtionalities. 
+This repository contains modules used for audit related functionalities. 
 
 ## Databases
 Refer to [SQL scripts](db_scripts).
 
 ## Build & run (for developers)
-The project requires JDK 1.11. 
+The project requires JDK 21.0.
```
Comment (dhanendra06) [inline README.md:13]: correct the java version

File: README.md [hunk 2]
```diff
@@ -23,7 +24,7 @@ The project requires JDK 1.11.
     ```
```
Comment (dhanendra06) [inline README.md:24]: Add the configuration

**Review by dhanendra06 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (GOKULRAJ136) [reply README.md:24]: added the configuration.

File: README.md (hunk 1 above)
Comment (GOKULRAJ136) [reply README.md]: corrected the java version to 21.0.3

**Review by dhanendra06 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**